### PR TITLE
✨ Add automated ad vendor import

### LINKED ISSUE #5322: Automate ad vendor list on amp.dev
### Problem
Currently, ad vendor update happens manually https://github.com/ampproject/amp.dev/blob/2c7be2541118b0d4832fcaa85c518db86b653322/pages/shared/data/faq.yaml#L73

### Solution
It could make sense to have them populated automatically so that we do not have to maintain the list any more.

### Additional information (optional)
An explanation of how users will be able to use the proposed feature, potentially including screenshots or designs.

### PR DESCRIPTION
**Pull request by lluerich**
* list of ad vendors will be fetched and injected during build
* fixes #5322

### CODE REVIEW

**Review by sebastianbenz — COMMENTED** [review]
Nice! Did you run the importer before the commit?

**Comment by lluerich** [comment]
Yeah, I did. I can commit the ```faq.yaml``` if you want.

**Comment by sebastianbenz** [comment]
Yes please, that'd be a good way to verify that it's working.

File: pages/shared/data/faq.yaml [hunk 1]
```diff
@@ -154,98 +180,130 @@ platform_and_vendor_partners:
...
       - label: Criteo
         url: https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/../../ads/vendors/criteo.md
-      - label: CSA
+      - label: Custom Search Ads
         url: https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/../../ads/vendors/csa.md
-      - label: CxenseDisplay
-        url: https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/../../ads/vendors/eas.md
-      - label: Dianomi
-        url: https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/../../ads/vendors/dianomi.md
+      - label: Custom (experimental)
```
Comment (sebastianbenz) [inline pages/shared/data/faq.yaml:217]: Can we filter this?

**Review by sebastianbenz — APPROVED** [review]
Great! Thanks! Looks like we've been missing quite a few! 

One comment otherwise LGTM

File: pages/shared/data/faq.yaml (hunk 1 above)
Comment (lluerich) [reply pages/shared/data/faq.yaml]: Yes, missed that one 😄

**Pull request merged**